src/Utility/Hash.php

```diff
@@ -138,9 +138,6 @@ public static function extract($data, $path)
         }
 
         if (strpos($path, '[') === false) {
-            if (function_exists('array_column') && preg_match('|^\{n\}\.(\w+)$|', $path, $matches)) {
```

Comment:
@ADmad Sorry that it is quite old PR, but this change increased 84% the performance for extracts in the `{n}.field` format. What do you think in keep it and just add a `is_array` check?

PS: The `function_exists` check can be eliminated now since we support 5.5+ only.

If you are ok, I can make a PR to re-add it with the proper check.